core/overlays/amun-api-stage/thoth-notification.yaml

```diff
@@ -0,0 +1,64 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: chat-notification-succeeded-
+  namespace: thoth-infra-stage
+  annotations:
+    argocd.argoproj.io/hook: PostSync
+    argocd.argoproj.io/hook-delete-policy: HookSucceeded
+spec:
+  template:
+    spec:
+      containers:
+        - name: chat-notification
+          image: registry.access.redhat.com/ubi8/ubi
+          command:
+            - "curl"
+            - "-X"
+            - "POST"
+            - "-H"
+            - "Content-Type: application/json; charset=UTF-8"
+            - "-d"
+            - "{'text':'I have successfully synchronized *Thoth Core* components to *STAGE* *amun-api* namespace, see <https://argocd-server-aicoe-argocd.apps.ocp.prod.psi.redhat.com/applications/stage-thoth-core-amun-api|ArgoCD UI> 🚀'}"
```

Comment:
this is amun

Reply:
*Thoth Core* should be *Amun API*

Reply:
isnt this correct we want core component to sync stage amun-api namespace?

Reply:
I think Amun API is not part of "Thoth Core" but is a standalone "application".

Reply:
oh yeah i should have made this standalone